### PR TITLE
Browser support

### PR DESCRIPTION
**Pull request by oriweingart**
Added gulp
Added bower
Supporting few browser for Chrome

### CODE REVIEW

File: dist/index.browser.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
+(function(window){
```
Comment (lgoldenberg) [inline dist/index.browser.js]: dist version should be minified

File: README.md [hunk 2]
```diff
@@ -3,10 +3,20 @@ Write fewer lines of code by turning node-style asynchronous functions and promi
 ## Requirements
 few is an npm module intended to run on node.js 4.0.0 and higher.  
 This package is continuously tested on all minor versions from node.js 4.0.0 and higher using Travis CI.
+
+Supported browsers:
+- Chrome 39
```
Comment (ikimia) [inline README.md]: Chrome 39 and up?
What about other browsers?

**Comment by ikimia** [comment]
I'm not familiar with compiling for bower.
But are all these files should be committed to the repo?
dist/index.browser.js
dist/index.browserify.js
dist/index.js

File: build/wrapper.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+(function(window){
+ 'use strict';
+  if (!window || window.few) {
+    throw "Few can't be registered to window.few";
+  }
+
+  // Monkey patch for process.nextTick
+  const process = {
```
Comment (ikimia) [inline build/wrapper.js]: Browserify should handle that.
https://github.com/substack/node-browserify#compatibility

File: build/wrapper.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    nextTick: function(func) {
+      return setTimeout(func, 0)
+    }
+  }
+  // Monkey patch for module.exports
+  var module = {
```
Comment (ikimia) [inline build/wrapper.js]: This is also handled by browserify using the `--standalone` flag.
http://www.forbeslindesay.co.uk/post/46324645400/standalone-browserify-builds

**Comment by ikimia** [comment]
Also, please use a descriptive name for the PR

**Comment by oriweingart** [comment]
1. Support all modern browsers using babel
2. Added minify version + source map to dist
3. Module is now bundled with browserify to support all load systems (require, AMD, window..)

**Pull request merged**